packages/lesswrong/components/posts/PostsTitle.tsx

```diff
@@ -66,7 +68,7 @@ const styles = (theme: ThemeType): JssStyles => ({
   leftCurated: {
     position: "relative",
     top: -1,
-    marginRight: 6
+    marginRight: theme.spacing.unit*0.5,
```

Comment:
I have made the styling so the start of the title text for pinned and curated posts line up